lib/rubocop/sketchup/cop/requirements/initialize_entity.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module SketchupRequirements
+      # SketchUp Entity objects (`Face`, `Edge`, `Group` etc) should not be
+      # initialized using the `new` constructor. These objects are managed by
+      # SketchUp. Instead use the `Sketchup::Entities#add_*` methods to create
+      # them.
+      class InitializeEntity < SketchUp::Cop
+
+        def_node_matcher :init_entity?, <<-PATTERN
+          (send
+            (const
+              (const nil? :Sketchup)
+              {:Face :Group :Edge :ConmponentInsstance :Image}
```

Comment:
`ConmponentInsstance` => `ComponentInstance`

Reply:
It's possible to call a method from a matcher. Syntax uses something like `#methodname` I think. But we could use that to make the node matcher look up a complete list of `Sketchup::Entity` derived classes.

Reply:
https://docs.rubocop.org/rubocop-ast/node_pattern.html#to-call-functions